README.md

```diff
@@ -171,7 +171,9 @@ However, because the Keychain is effectively disk storage, there is no guarantee
 
 Already using the Keychain and no longer want to maintain your own Keychain code? We feel you. That’s why we wrote `migrateObjects(matching query: [String : AnyHashable], removeOnCompletion: Bool)`. This method allows you to migrate all your existing Keychain entries to a Valet instance in one line. Just pass in a Dictionary with the `kSecClass`, `kSecAttrService`, and any other `kSecAttr*` attributes you use – we’ll migrate the data for you.
 
-### Supporting macOS Catalina (10.15) and later
+### Integrating Valet into a macOS application
+
+Your macOS application must have the Keychain Sharing entitlement in order to use Valet, even if your application does not intend to share keychain data between applications. For instructions on how to add a Keychain Sharing entitlement to your application, read [Apple's documentation on the subject](https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps). The subsection on "Add Apps to One or More Keychain Access Groups" is particularly helpful, as it includes a screenshot showing Xcode's Signing & Capabilities screen. For more information on why this requirement exists, see [issue #213](https://github.com/square/Valet/issues/213).
```

Comment:
You can use https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps#2974918 to link directly to the "Add Apps to One or More Keychain Access Groups"` section

Reply:
That's way better! When you have a chance, I'd love to know how you generated that link. I tried to find link anchors on that page but came up short.

Reply:
I just inspected the HTML 😁 


![image](https://user-images.githubusercontent.com/530662/73219861-2ee50800-41b1-11ea-9ca3-84728e25e741.png)

Reply:
Smart!! I'm going to have to steal that trick.